### ISSUE TITLE
Possible bias in training

### ISSUE THREAD

**Issue by Lucashsmello** [uni-medical/SAM-Med3D#70]
Hi! Thanks a lot for the code!

Does using `torchio.CropOrPad(mask_name='label')` in the training phase not contribute to a bias? Because of always centering the image on the label.

**Comment by blueyo0** [comment]
Indeed, your observation is correct. This version is designed with an excessive focus on the center. While this approach proves detrimental for targets exceeding 128 pixels, it has negligible impact on small to medium-sized targets. 

If your focus is large targets, further fine-tuning is needed.
